alvr/client_mock/src/main.rs

```diff
@@ -189,6 +189,7 @@ fn tracking_thread(
                 fb_face_expression: None,
                 htc_eye_expression: None,
                 htc_lip_expression: None,
+                pico_face_expression: None,
```

Comment:
As discussed on Discord, this is a breaking change and so we should reuse the fb_face_expression field for pico.

Reply:
Shall I rename it into `face_expression`?

Reply:
While it wouldn't break the protocol (because it's binary and packed) i think we should keep the current name for now. We will add a proper separate field on v21.